ashritha_shetty/sum_range.cpp

```diff
@@ -0,0 +1,20 @@
+/*
+Created on Novemberer 08, 2018
+@author: ashritha27
+*/
+//Program to find the sum of numbers in a given range in C++
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    int min,max,sum=0;
```

Comment:
Make each of these into blocks and leave a comment for each block. Like for this one, it can be 

// Declaring the range variables and Intializing the result variable